openeo/metadata.py

```diff
@@ -553,8 +569,8 @@ def get_band_names(bands: List[Band]) -> List[str]:
     def is_band_asset(asset: pystac.Asset) -> bool:
         return "eo:bands" in asset.extra_fields
 
+    temporal_dimension = []
```

Comment:
maybe I'm misunderstanding something, but using an empty list as default value for a dimension object looks weird